### PR TITLE
Update documentation "npm"  to "yarn"

### PR DESCRIPTION
**Pull request by Stroller15**
Update documentation "npm"  to "yarn"

issue solved #89

### CODE REVIEW

**Comment by Stroller15** [comment]
@himanshu1221 can you my PR and suggest if there is anything wrong

File: README.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ Check out the website: [here](https://www.vibey.live/)
...
-2. Clone the repository `git clone https://github.com/<USERNAME>/vibey`
-3. Run `cd vibey` to change the working directory
-4. Run `npm ci` or `npm clean-install` to install all dependencies
-5. Run `npm run dev` to start the application
-6. Visit <https://localhost:3000> to view the application
-
+1.Fork the repo
+2.Clone the repository git clone https://github.com/<USERNAME>/vibey
+3.Run cd vibey to change the working directory
+4.Run yarn install or yarn to install all dependencies
```
Comment (Deepu178) [inline README.md:33]: 4.Run yarn install or yarn to install all dependencies
change this to 
4.Run `yarn install` or `yarn` to install all dependencies

File: README.md [hunk 2]
```diff
@@ -23,17 +23,17 @@ Check out the website: [here](https://www.vibey.live/)
...
+5.Run yarn dev to start the application
```
Comment (Deepu178) [inline README.md:34]: change this: 
5.Run yarn dev to start the application
to
5.Run `yarn dev` to start the application

**Review by Deepu178 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Stroller15) [reply README.md:33]: Hi @Deepu178 I have to again raise PR with this changes or how to edit this

Comment (Stroller15) [reply README.md:33]: can you help out with this

Comment (Deepu178) [reply README.md:33]: you just need to commit the changes and no need to create a new pr @Stroller15

**Pull request closed without merging**